src/fields/abstractField.js

```diff
@@ -5,6 +5,9 @@ function checkIfValidatorIsAStringAndConvertToObject(validator) {
 	if(isString(transformedValidator)){
 		let chunks = transformedValidator.split(".");
 		let root = chunks.shift();
+		if(typeof window[root] == "undefined"){
+			throw "[VueFormGenerator] window["+root+"] is not defined. For using dot notation json strings in validators, look at the documentation.";
```

Comment:
I would prefer es6 string interpolation as it looks better and more clear. 
throw `[VueFormGenerator] window[${root}] is not defined. If you want to use strings when defining a validator please see link here`